indent/verilog_systemverilog.vim

```diff
@@ -65,16 +65,16 @@ if exists("*GetVerilogSystemVerilogIndent")
   finish
 endif
 
-if exists("g:verilog_disable_indent")
-  let s:verilog_disable_indent = split(g:verilog_disable_indent, ",")
-else
-  let s:verilog_disable_indent = []
-endif
-
 set cpo-=C
 
 function! GetVerilogSystemVerilogIndent()
 
+  if exists("g:verilog_disable_indent")
+    let s:verilog_disable_indent = split(g:verilog_disable_indent, ",")
+  else
+    let s:verilog_disable_indent = []
+  endif
+
```

Comment:
As we are doing similar things, perhaps we postpone doing anything extra until another time.

The performance difference is probably in the range of 0.5-2% which is tiny so probably not worth worrying about for now.